src/psyclone/tests/psyir/transformations/extract_test.py

```diff
@@ -46,6 +46,7 @@
 from psyclone.domain.lfric.transformations import LFRicExtractTrans
```

Comment:
I've realised this file does not full cover `extract_trans.py`:

    PSyclone/src/psyclone/psyir/transformations/extract_trans.py      34     12    65%   96, 169-208

Also, it should be named `extract_trans_test.py`. However, this is already a large PR so I think this can be fixed at a later date (and also perhaps the tests for ExtractNode separated into the correct location).

Reply:
I've covered all lines, but didn't do the rename (I think that would make the review harder). I agree, I should fix this.

Reply:
Great, thanks.